src/applications/simple-forms/21-4138/components/breadcrumbs.jsx

```diff
@@ -1,28 +1,28 @@
-import { VaBreadcrumbs } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
 import React from 'react';
 
-export const Breadcrumbs = (
-  <VaBreadcrumbs
-    className="breadcrumbs-container"
-    breadcrumbList={[
-      {
-        href: '/',
-        label: 'Home',
-      },
-      {
-        href: '/supporting-forms-for-claims',
-        label: 'Supporting forms for VA claims',
-      },
-      {
-        href:
-          '/supporting-forms-for-claims/statement-to-support-claim-form-21-4138',
-        label: 'Submit a statement to support a claim',
-      },
-    ]}
-    label="Breadcrumb"
-    homeVeteransAffairs={false}
-  />
-);
+export const Breadcrumbs = () => {
+  const breadcrumbs = [
+    { href: '/', label: 'Home' },
+    {
+      href: '/supporting-forms-for-claims',
+      label: 'Supporting forms for VA claims',
+    },
+    {
+      href:
+        '/supporting-forms-for-claims/statement-to-support-claim-form-21-4138',
+      label: 'Submit a statement to support a claim',
+    },
+  ];
+  const bcString = JSON.stringify(breadcrumbs);
+  return (
+    <va-breadcrumbs
```

Comment:
Just for curiosity - Is the old approach still OK as well? I assume `VaBreadcrumbs` are also on v3 now? or is `va-breadcrumbs` doing something special here that `VaBreadcrumbs` can't handle? Just thinking, as a consumer, if I had the preference I would prefer not to stringify my params unless necessary.

On another note, I just introduced some new code using `VaBreadcrumbs` at `simple-forms/mock-form-minimal-header/components/breadcrumbs.jsx` today. I can adjust that as a followup if this is the new recommendation.

Reply:
I believe using the react-binding version - VaBreadcrumbs - is still supported. I was just converting everything to use the web component version - va-breadcrumbs - and standardized on that.